components/brave_rewards/resources/android_page/components/tipsBox.tsx

```diff
@@ -79,6 +79,14 @@ class TipBox extends React.Component<Props, State> {
     })
   }
 
+  getDescriptionChild = () => {
```

Comment:
Nit: I think `getDescription` would be fine here.

Reply:
Changed in 6231508161154fd72b57caa05f0fa40bf641f429